package/MDAnalysis/analysis/MSD.py

```diff
@@ -0,0 +1,468 @@
+# -*- Mode: python; tab-width: 4; indent-tabs-mode:nil; coding:utf-8 -*-
+# vim: tabstop=4 expandtab shiftwidth=4 softtabstop=4
+#
+# MDAnalysis --- http://www.MDAnalysis.org
+# Copyright (c) 2006-2015 Naveen Michaud-Agrawal, Elizabeth J. Denning, Oliver Beckstein
+# and contributors (see AUTHORS for the full list)
+#
+# Released under the GNU Public Licence, v2 or any higher version
+#
+# Please cite your use of MDAnalysis in published work:
+#
+# N. Michaud-Agrawal, E. J. Denning, T. B. Woolf, and O. Beckstein.
+# MDAnalysis: A Toolkit for the Analysis of Molecular Dynamics Simulations.
+# J. Comput. Chem. 32 (2011), 2319--2327, doi:10.1002/jcc.21787
+#
+
+"""
+MSD --- :mod:'MDAnalysis.analysis.MSD'
+==============================================================
+
+:Author: Andrew Biedermann
+:Year: 2016
+:Copyright: GNU Public License v3
```

Comment:
Sorry, I thought 'GNU Public License v3' would be GPLv2 and higher. I will change it to 'GNU Public License v2.